test/FunctionalTestUtils/project.json

```diff
@@ -1,18 +1,19 @@
 {
+  "version": "1.0.1",
```

Comment:
Why it wasn't here before and why we don't have it for the main package's `project.json`? How the version of our NuGet is defined?